spec/fixtures/json/patients/CMS108v7/INR4no_decimal_DayOfAnes_NUMERPass.json

```diff
@@ -0,0 +1,404 @@
+{
```

Comment:
This patient should be added to bonnie-fixtures to make the re-generation of this fixture easier